### DISCUSSION TITLE
Preview 1.13.10734.0?

### DISCUSSION THREAD

**Discussion by rexbinary** [microsoft/terminal#12744] [General]
![image](https://user-images.githubusercontent.com/740274/159832556-1d4864c3-ee83-4505-9459-fd9c3baebe6a.png)

I just received this from the Microsoft Store, but I don't see it here? Is this legit?

**Comment by DHowett** [comment]
It's legit! I'll get the release notes for the 1073 series out on Thursday (Central US time).

**Reply by DHowett** [reply]
Since we published the release notes, I'll mark this one as answered/closed. Thanks!
